src/main/java/gnu/io/NativeResource.java

```diff
@@ -186,7 +186,9 @@ private InputStream locateResource(String name) {
 				}
 			}
 		}else if(OSUtil.isFreeBSD()) {
-			if(OSUtil.is64Bit()) {
+			if(OSUtil.isARM() && OSUtil.is64Bit()) {
```

Comment:
Thank you for refactoring this condition. I know it's stylistically at odds with the other platform conditions, but IMO it reads better for a platform with only three options. I'd like to replace these condition trees with a table-based lookup anyway.